.gitignore

```diff
@@ -29,3 +29,5 @@ hail-*.log
 
 # vscode settings
 **/.vscode/
+
+graphql-api/unversioned_config.json
```

Comment:
**nitpick (non-blocking)**: It looks like the other env/config type files bundled into various dockerfiles starting at line 16, maybe this is a better fit to put in there?